code/game/objects/items/weaponry.dm

```diff
@@ -644,6 +644,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = 12
 	wound_bonus = -10
 	throwforce = 12
+	demolition_mod = 1.5
```

Comment:
Good point.
I've definitely taken into consideration things like windows and other high integrity items, I think it makes sense that a baseball bat would be better at smashing in a window than a scalpel of the same force. Of course, if there are any instances in specific that you think are too strong/too weak, its not a deal breaker to change or remove the modifiers.

Looking at specific thresholds:
Normal Airlocks have a deflection of 21, no items changed would reach this number with the possible exception of His Grace at a certain number of souls. Notably, fireaxes can now break down reinforced airlocks.
Lockers have deflection of 20, which again means they will be able to be broken by His Grace with a few amount of souls, but otherwise are unaffected.
Also notably, picked up chairs will be strong enough to break r-windows.

Tell me if anything worries you, and I can change it.

Reply:
Additionally, you can no longer break down r-windows with sharpened knives, scalpels, captain's saber, and the curator's whip. Which was the inspiration for this PR in the first place.